spec/amp-html-format.md

```diff
@@ -195,7 +195,7 @@ HTML tags can be used unchanged in AMP HTML. Certain tags have equivalent custom
   </tr>
   <tr>
     <td width="30%">script</td>
-    <td>Prohibited unless the type is <code>application/ld+json</code>, <code>application/json</code>, or <code>text/plain</code>. (Other non-executable values may be added as needed.) Exception is the mandatory script tag to load the AMP runtime and the script tags to load extended components.</td>
+    <td>Prohibited unless the type is <code>application/ld+json</code>, <code>application/json</code>, <code>text/plain</code>, or <code>application/octet-stream</code>. (Other non-executable values may be added as needed.) Exception is the mandatory script tag to load the AMP runtime and the script tags to load extended components.</td>
```

Comment:
this is technically true, but only in a very limited context. It must be a [subscriptions](https://newsinitiative.withgoogle.com/subscriptions) block with encrypted contents. Nobody will be hand writing these blocks, they will be generated by code.

I don't think it's necessary to mention it here, as it's not generally usable for arbitrary use, but I don't feel strongly either way.